CONTRIBUTING.md

```diff
@@ -30,7 +30,7 @@ Run `/opt/bin/deis-debug-logs` on one of your CoreOS machines and provide the ou
 ## Getting Started
 
 - Fork the repository on GitHub
-- Read the README.md for build instructions
+- Read the `hacking.rst` for build instructions
```

Comment:
Would you mind changing this to "Read the documentation for build instructions", with "the documentation" linked to http://docs.deis.io/en/latest/contributing/hacking/? `hacking.rst` is not a top-level document so users will get confused where to find this. It's better to point to the official documentation, anyways (which hosts this document).

Thanks!